### DISCUSSION TITLE
Is Bisq supporting illegal money traders by not allowing to penalize their accounts?

### DISCUSSION THREAD

**Discussion by slrslr** [bisq-network/bisq#6990] [Q&A]
I was told that there is no way in Bisq to penalize an account which caused sending money that were later (after Bisq trade is finalized) confiscated because it was result of an illegal activity.

**Comment by pazza83** [comment]
If you report then the user would be banned from trading on Bisq using their onion address and the account details used. if you had any other trades with them you could take them to mediation / arbitration and the user would lose 100% of the funds.

Have you had the situation happen to you? It is a very rare occurrence on Bisq

**Reply by slrslr** [reply]
Thank you for all parts of your posts.
Though i wish that you mention on how to 1) report and 2) take them to mediation / arbitration in Bisq after trade with them is finalized. Wiki fails to explain it too, at least I am unable to find it at https://bisq.wiki/index.php?search=report

**Reply by pazza83** [reply]
You can reach me on Matrix to report any issues: https://bisq.wiki/Matrix_bisq.chat

My username there is: @pazza83:matrix.org

**Reply by marcovelon** [reply]
> @pazza83: [...] if you had any other trades with them you could take them to mediation / arbitration and the user would lose 100% of the funds.

Does this mean that if a BTC buyer has an unfinished trade and determines that the seller's BTC come from an illicit source confirmed by any third-party AML screening service, the buyer can open a mediation/arbitration and make the BTC seller lose 100% of their funds?

If so, what will happen to these funds? 

I would appreciate if you could provide a public answer here, since the Bisq users need to understand and be acknowledged on your current workflow in regards to such situations.

**Reply by pazza83** [reply]
Hi @marcovelon 

My quoted comments refer to fiat rather than bitcoin. Ie if a chargeback scammer was doing chargebacks post payout then the scammer could be punished in subsequent trades. 

In the case you mention the buyer could go though the dispute process;

- Speak to seller to see if can be resolved (for example buyer and seller agree to cancel the trade)
- Open mediation
- Open arbitration
- Escalate case to the Bisq DAO

The funds are held in a 2 of 2 multisig so ultimately what happens to the funds is down to what is agreed upon by the buyer and seller. If no agreement could be reached then the trade can be sent to arbitration.
